node/system_function.go

```diff
@@ -0,0 +1,71 @@
+package node
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/buildpack/libbuildpack/layers"
```

Comment:
base on @jabrown85's comment above, should this be `github.com/heroku/libhkbuildpack`?

Reply:
Same response as above: https://github.com/heroku/node-function-buildpack/pull/23/files#r312633837.